packages/core/src/components/TermList/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import withStyles, { WithStylesProps } from '../../composers/withStyles';
+import Term from './Term';
+
+export { Term };
+
+function TermList({
+  inline,
+  children,
+  cx,
+  styles,
+}: {
+  link?: NonNullable<React.ReactNode>;
+  endAlign?: boolean;
+  inline?: boolean;
+  keyTextRegular?: boolean;
+  children: NonNullable<React.ReactNode>;
+} & WithStylesProps) {
+  return <dl className={cx(inline && styles.inline, styles.termList)}>{children}</dl>;
```

Comment:
aah, so this would be `styles.termList, inline && styles.termList_inline` to follow our modifier pattern more